applications/Xcavate.md

```diff
@@ -0,0 +1,171 @@
+
+![](https://xcavate.io/W3F/DeveloperLoanProcessPOC.pdf)
+
+Xcavate will adapt and build multiple Pallets to execute the functionalities. We will use the KILT SDK as well as the substrate framework and primary language Rust to create the pallets on the Polkadot ecosystem.
+Token Pallet
+Decentralized Identity Document pallet
+User Management pallet
+Loan Management pallet
+Transaction management pallet
+Staking pallet
+Verification pallet
+
+This Architecture diagram shows the visual representation that maps out the physical implementation for components in the Polkadot ecosystem and how the scalability of the platform works. The diagram represents Microservices we are using to execute the functionality. API gateway will split the API calls to the respective microservices. We are using an s3 bucket to store the media files. We are using pallets to execute the functions on the Polkadot ecosystem.
+
+![](https://xcavate.io/W3F/XCAV-WEB-3-Grant-Application-Image-4.jpg)
+
+Token:
+XCAV - Platforms Native Token to launch a para chain and participate in a crowd loan/slot auction. XCAV tokens are used to stake within the platform. We will create these tokens with the Transfer, Mint, Burn, Approve, and Allowance functions.
+Liquidity Pool:
+Xcavate creates a liquidity pool to store staked tokens, staked tokens will be queued and utilized for each loan application. XCAV tokens are swapped as GBPT through XCM and deposited on the Central Loan pool.
+Central Loan Pool:
+Xcavate Creates a central loan pool to manage Loan amounts and Loan repayment. The central lending pool will act as the heart of the platform to map all the transactions.
+
+### Technology Stack
+
+React.js
+Node.js
+Express.js
+Rust
+Substrate Framework
+Polkadot
+
+### Third-Party
+
+Kilt – SDK
+Chainlink
+AWS
+
+### Ecosystem Fit
+
+After many months of research, we initially met on a zoom with Ingo from Kilt, discussing our idea and talking through how the Kilt chain can be incorporated into the Xcavate system. A few weeks later, we contacted Acala and met with Robinson, who was very helpful and enthusiastic about us being involved in the ecosystem and our potential partnership. We also reached Centrifuge and spent many weeks looking through their docs with the initial idea of creating a dApp on their chain, but as there were some striking differences in how their chain operated, it was apparent that was not going to be the best fit for our project. Going forward, however, once the real estate has been built, there will be opportunities for us to introduce Centrifuge.
+
+We feel this is a great way to bring land & Property assets into the digital ecosystem to interact with the following potential partners:

```

Comment:
Are these actual partnerships, or is this your tech stack?

Reply:
KILT, Sub Wallet (in talks with them), Nova Wallet & Bittensor (need to start a conversation with them) would be partnerships... I suppose Chainlink & Crust would just be tech stack